src/functionalTest/java/uk/gov/hmcts/probate/functional/fee/SolCcdServiceFeeTests.java

```diff
@@ -27,7 +27,7 @@
 public class SolCcdServiceFeeTests extends IntegrationTestBase {
 
     private static final int APP_FEE = 27300; //comment this out for local tests - keep for commits
-    //private static final int APP_FEE = 15500;
+    // private static final int APP_FEE = 15500;
```

Comment:
can we delete this commented code

Reply:
It meant to be keep, as for local env we have to change to 15500 as per Sanjay's change